pkg/storegateway/bucket_index_reader.go

```diff
@@ -572,13 +569,23 @@ func newBucketIndexLoadedSeries() *bucketIndexLoadedSeries {
 	}
 }
 
-// loadSeriesForTime populates the given symbolized labels for the series identified by the reference if at least one chunk is within
+// addSeries stores a series raw data in the data structure. A stored series can be loaded via unsafeLoadSeriesForTime().
```

Comment:
Do you mean it's the raw data of a series?
```suggestion
// addSeries stores a series' raw data in the data structure. A stored series can be loaded via unsafeLoadSeriesForTime().
```